--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,7 +6,7 @@
 OPTION(BITCOIN_ENABLE_OPENCL "Enable OpenCL miner" OFF)
 OPTION(BITCOIN_ENABLE_REMOTE_SERVER "Enable remote miner server" ON)
 OPTION(BITCOIN_BUILD_GUI "Build GUI (bitcoin)" OFF)
-OPTION(BITCOIN_BUILD_DAEMON "Build Daemon (bitcoind)" ON)
+OPTION(BITCOIN_BUILD_DAEMON "Build Daemon (bitcoind)" OFF)
 OPTION(BITCOIN_BUILD_REMOTE_MINER "Build remote miner (bitcoinr)" ON)
 
 SET(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake-modules/" ${CMAKE_MODULE_PATH})
